addons/l10n_latam_check/models/account_payment.py

```diff
@@ -93,6 +93,9 @@ def _compute_check_number(self):
         from_checkbooks = self.filtered(lambda x: x.l10n_latam_checkbook_id)
         third_party_checks = self.filtered(lambda x: x.payment_method_line_id.code == 'new_third_party_checks')
         for pay in from_checkbooks:
+            # we don't recompute when creating from a method and if check_number is sent
+            if pay.check_number and not isinstance(pay.id, models.NewId):
```

Comment:
@jjscarafia discussing with @jco-odoo doing the "on change" it will be better than doing this. Can you help e with this change so I can test

Reply:
already change, I can't mark as solved the comments